### ISSUE TITLE
scenarios are not persisted correctly to/read from XLS template

### ISSUE THREAD

**Issue by okworx** [GreenDelta/epd-editor#104]
**Test case:** 
- create new EPD dataset
- choose EF3.1 profile
- create scenarios "100% incineration" and "100% recycling", leaving group empty, selecting the former as default 
![CleanShot 2024-10-14 at 17 52 23@2x](https://github.com/user-attachments/assets/5e98349b-e02e-40a6-acd2-5d6c5f93d7dd)

- add modules for both scenarios
![CleanShot 2024-10-14 at 17 53 19@2x](https://github.com/user-attachments/assets/fafe82e6-030f-46b5-9fb9-0c46e0a1ea43)

- synchronize results table
- export results table to XLS template
- add some test numbers to XLS template: 
[results_EF3.1_dual_scenario.xlsx](https://github.com/user-attachments/files/17366507/results_EF3.1_dual_scenario.xlsx)
- create new EPD dataset
- import XLS template

**Observed behavior:**

- there are duplicate entries in the scenarios list
- the default status is not restored 
![CleanShot 2024-10-14 at 17 57 23@2x](https://github.com/user-attachments/assets/ea844085-e777-4b8b-8cab-7ed6bf6bf7c6)

**Expected behavior:**
- only two scenarios are in the scenarios list
- the default status is restored ("100% incineration" marked as default)
- bonus: the correct profile is being picked up and reflected in the profile drop-down

The same behavior can be observed when explicitly assigning a group ID. When adding another scenario group and using explicit group IDs, the group IDs and default status is lost as well (example XLS template: [results_EF3.1_quad_scenario.xlsx](https://github.com/user-attachments/files/17366578/results_EF3.1_quad_scenario.xlsx))

In general, all declared scenarios without a group ID should be treated as belonging to the same group.

**Comment by msrocka** [comment]
@okworx  I created a pre-release where I tried to solve this: https://github.com/GreenDelta/epd-editor/releases/tag/v6.0.3

**Comment by okworx** [comment]
> @okworx I created a pre-release where I tried to solve this: https://github.com/GreenDelta/epd-editor/releases/tag/v6.0.3

Awesome, thanks! Will there be a macOS build too for testing?

**Comment by msrocka** [comment]
@francoislerall could you make a test build for macOS?

**Comment by francoislerall** [comment]
> @francoislerall could you make a test build for macOS?

Yes, today or tomorrow

**Comment by francoislerall** [comment]
I have added a macOS build :)

**Comment by okworx** [comment]
Thank you guys for the fix and the new build!

While this works in principle, it will not work anymore when people use slashes for scenario names, which eventually someone will. Consider this case:

![Image](https://github.com/user-attachments/assets/ad82adec-b34e-4c75-a08a-5d38fc77146a)

Exporting and re-importing the XLSX template for this example will not work. 

Therefore, I suggest using something as a separator which is more unlikely to be used in names. How about three slashes, for example `///`? Or maybe you've got a better idea.

**Comment by msrocka** [comment]
True, an alternative would be to only split the `{module} / {scenario}` identifier once after the first `/` for now.

**Comment by okworx** [comment]
Is there any chance to implement this in order to have this issue fully fixed and publish a new release?

**Comment by msrocka** [comment]
will be fixed in v7
